about/package-scope.md

```diff
@@ -30,9 +28,229 @@ open a [pre-submission inquiry using a GitHub Issue](https://github.com/pyOpenSc
 one of our editors. We are happy to look at your package and help you understand
 whether it is in scope or not.
 
-```{include} /appendices/scope.md
+## About the scope of packages that we review
+
+pyOpenSci supports packages that support both open reproducible science,
+data processing and and the various stages of managing the
+data lifecycle. Packages submitted to pyOpenSci should fit into one or
+more of the categories below.
+
+```{admonition} Package Use Metrics Are Not a Requirement for Review
+:class: important
+pyOpenSci has no requirements surrounding the volume community use of your
+package.
+If you package is well developed, the number of GitHub stars or PyPI or conda
+downloads is NOT considered as a part of our scope evaluation.
+
+We welcome young packages that are just entering the scientific Python
+ecosystem to apply for review if they are relevant to the science community and
+fit into at least one scope category below.
 ```
```

Comment:
awesome!! thanks!! i realized last night that we probably want to mention that we also welcome more mature packages. otherwise we will get the question " our question is mature with a growing user base, are we in scope?" :)